futurex_openedx_extensions/helpers/export_data.py

```diff
@@ -0,0 +1,69 @@
+"""
+This module contains a mixin for exporting data to CSV format and related helpers.
+"""
+import csv
+import os
+
+from django.conf import settings
+
+
+class ExportCSVMixin:
+    """
+    Mixin for exporting data to CSV format.
+    """
+    @property
+    def csv_header(self) -> list:
+        """Get the headers from the serializer fields."""
+        return list(self.get_serializer().fields.keys())  # type: ignore[attr-defined]
+
+    @property
+    def csv_data(self) -> dict:
+        """Get the serialized data"""
+        queryset = self.get_queryset()  # type: ignore[attr-defined]
```

Comment:
we can do this for now, but the next step would be managing data export as like as pagination. A chunk of data after another